### DISCUSSION TITLE
Organization stats

### DISCUSSION THREAD

**Discussion by camipozas** [anuraghazra/github-readme-stats#1808] [Q&A] [answered]
If I work in an organization, can I add statistics to my readme?

**Comment by rickstaa** [comment, marked as answer]
@camipozas Currently, we do not include stats of organization repositories in the cards. However, this is one of the most requested features (see #1). This feature has been implemented in #1122, but some things still must be fixed. However, you can already use this feature on your own fork by deploying your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance). You can follow the steps explained in https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 to do this.

**Reply by rickstaa** [reply]
@camipozas Did that solve your question?

**Reply by camipozas** [reply]
sorry for the delay, what is the query to add an organization? or url, I didn't find it

**Reply by rickstaa** [reply]
Great, could you maybe accept my answer so that this question shows up as answered? 🤓

**Reply by camipozas** [reply]
yes, but I still don't understand if I forked the repository, I did the deploy in vercel and everything... how my organization's data looks, I see it as it was without them. Do I have to add any query? thanks

**Reply by rickstaa** [reply]
@camipozas You should add the `&role=OWNER,COLLABORATOR` parameter.

**Reply by camipozas** [reply]
@rickstaa thank you but I'm using this url `https://github-stats.cam1pozas.xyz/api?username=camipozas&role=OWNER,COLLABORATOR` but says:
```
Something went wrong! file an issue at https://tiny.one/readme-stats
Parse error on "<" (error) at [25, 1] 
Make sure the provided username is not an organization
```

**Reply by rickstaa** [reply]
I think conflicts with the master branch caused that. I just updated the https://github.com/anuraghazra/github-readme-stats/pull/1122 such that it does not conflict with the master branch. I checked it on my branch, and now it works (see https://github.com/rickstaa/github-readme-stats/tree/master-rstaa).

**Comment by Nik6511** [comment]
I don't think that feature is available right now but I also want to have it on my profile<

**Reply by rickstaa** [reply]
As stated above, this feature has indeed not yet been released. You can already use this feature by following the steps explained in https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 to do this.

**Reply by Nik6511** [reply]
Thankyou!

**Comment by domdfcoding** [comment]
I assume the status hasn't changed as of now (August 2025)? I tried it and it doesn't work, but I assume the feature still isn't ready to merge?
